solutions/observability/apm/configure-logstash-output.md

```diff
@@ -218,7 +218,7 @@ The `proxy_use_local_resolver` option determines if {{ls}} hostnames are resolve
 
 #### `index` [apm-logstash-index]
 
-The index root name to write events to. The default is `apm-server`. For example `"apm"` generates `"[apm-]VERSION-YYYY.MM.DD"` indices (for example, _"apm-{{version}}-2017.04.26"_).
+The index root name to write events to. The default is `apm-server`. For example `"apm"` generates `"[apm-]VERSION-YYYY.MM.DD"` indices (for example, _"apm-{{stack-version}}-2017.04.26"_).
```

Comment:
Since this is just an example we probably don't _have_ to use a version variable, but in my opinion it doesn't hurt.